api/main.py

```diff
@@ -7,18 +7,24 @@
 from fastapi import FastAPI
 
 from api.routes.v1.config import router as config_router
+from api.routes.v1.sic_lookup import router as sic_lookup_router
 
 app: FastAPI = FastAPI(
-    title="LLM API",
-    description="API for interacting with LLM",
-    version="1.0",
+    title="Survey Assist API",
+    description="API for interacting with backend data processing services such as classification",
+    version="1.0.0",
+    contact={"name": "ONS Digital", "email": "steve.gibbard@ons.gov.uk"},
```

Comment:
Is the contact info necessary for API docs? We have organisational policies that recommend not making ONS emails public where you can (I am actually ok with my email being public, but an FYI).